tensorboard/plugins/graph/graphs_plugin_v2_test.py

```diff
@@ -65,17 +65,17 @@ def _get_graph(self, plugin, *args, **kwargs):
         self.assertEqual(mime_type, "text/x-protobuf")
         return text_format.Parse(graph_pbtxt, graph_pb2.GraphDef())
 
-    @graphs_plugin_test.with_runs(
-        [
-            graphs_plugin_test._RUN_WITH_GRAPH_WITH_METADATA,
-            graphs_plugin_test._RUN_WITHOUT_GRAPH_WITH_METADATA,
-        ]
-    )
-    def test_info(self, plugin):
+    def test_info(self):
         raise self.skipTest(
             "TODO: enable this after tf-nightly writes a conceptual graph."
         )
```

Comment:
dang, this is an old TODO that I can remove.

Reply:
Yep. :-) am considering improving these tests while I’m here, both to
cover more classes of data and to use the public APIs rather than the
internals. If so, I’ll poke at this.